plugins/gcloud/gcloud.plugin.sh

```diff
@@ -0,0 +1,49 @@
+#! bash oh-my-bash.module
+# gcloud.plugin.sh
+# Author: Antonino Cangialosi (github.com/ninoCan)
+# Fork of oh-my-bash gcloud plugin
+
+if [[ -z "${CLOUDSDK_HOME}" ]]; then
+    search_locations=(
+        "$HOME/google-cloud-sdk"
+        "/usr/local/Caskroom/google-cloud-sdk/latest/google-cloud-sdk"
+        "/opt/homebrew/Caskroom/google-cloud-sdk/latest/google-cloud-sdk"
+        "/usr/share/google-cloud-sdk"
+        "/snap/google-cloud-sdk/current"
+        "/usr/lib/google-cloud-sdk"
+        "/usr/lib64/google-cloud-sdk"
+        "/opt/google-cloud-sdk"
+        "/opt/local/libexec/google-cloud-sdk"
+    )
+
+    for gcloud_sdk_location in $search_locations; do
+        if [[ -d "${gcloud_sdk_location}" ]]; then
+            CLOUDSDK_HOME="${gcloud_sdk_location}"
+            break
+        fi
+    done
+    unset search_locations gcloud_sdk_location
```

Comment:
Could you put the entire code in a shell function and declare these variables as `local` so that we do not affect the global variables? FWIW, the naming convention #280 suggests the function name of the form `_omb_plugin_gloud_xxxxx` (such as `_omb_plugin_gloud_initialize`).

In that way, we do not need to unset the used variables and also can avoid possible conflicts with the global variables of the same names defined in other places. Or otherwise, please rename these temporary global variables to have the names of `_omb_plugin_gloud_xxxx` (#280).

Reply:
I will asap!